src/Arcus.Templates.WebApi/Program.cs

```diff
@@ -25,12 +25,13 @@ public static IWebHostBuilder CreateWebHostBuilder(string[] args)
             string httpEndpointUrl = "http://+:" + configuration["ARCUS_HTTP_PORT"];
             IWebHostBuilder webHostBuilder =
                 WebHost.CreateDefaultBuilder(args)
-                       .UseConfiguration(configuration)
-                       .UseUrls(httpEndpointUrl)
-                       .ConfigureLogging(logging => logging.AddConsole())
-                       .UseStartup<Startup>();
+                    .ConfigureKestrel(kestrelServerOptions => kestrelServerOptions.AddServerHeader = false)
```

Comment:
Hmm, change in formatting?

Reply:
Ah must be my R# settings at work, reverting.